### PR TITLE
Added more information to the PST tables

### PR DESCRIPTION
**Pull request by piuswalter**

### CODE REVIEW

File: src/Exercise03AI.ipynb [hunk 1]
```diff
@@ -42,11 +42,9 @@
...
     "\n",
-    "Anhand der Piece-Square Table der Bauern `PST_PAWN`, die im Folgenden definiert ist, wird ersichtlich, dass Bauern auf den Feldern `f2`, `g2` und `h2` (symmetrischer Weise natürlich auch auf `a2`, `b2` und `c2`) Boni (positive Bewertung) erhalten, während sie auf den Felder `f3` und `g3` Abzüge (negative Bewertungen) erhalten.\n",
-    "\n",
-    "Das Dictionary `PIECE_TO_PST` weist jedem Figurentyp die entsprechende umgedrehte Piece-Square Table zu, welche aufgrund der einfacheren Lesbarkeit in umgekehrter Reihenfolge initialisiert werden. Um für die Figur des Königs zwischen Mittel- und Endpiel unterscheiden zu können, existiert zusätzlich das Dictionary `PIECE_TO_PST_KING_ENDGAME`.\n",
+    "Das Dictionary `PIECE_TO_PST` weist jedem Figurentyp die entsprechende an der x-Achse gespiegelte Piece-Square Table mithilfe der Funktion `mirror_pst` zu, welche aufgrund der einfacheren Lesbarkeit in umgekehrter Reihenfolge initialisiert wird. Um für die Figur des Königs zwischen Mittel- und Endpiel unterscheiden zu können, existiert zusätzlich das Dictionary `PIECE_TO_PST_KING_ENDGAME`.\n",
```
Comment (p-fruck) [inline src/Exercise03AI.ipynb:47]:
```suggestion
    "Das Dictionary `PIECE_TO_PST` weist jedem Figurentyp die entsprechende, an der x-Achse gespiegelte, Piece-Square Table mithilfe der Funktion `mirror_pst` zu, welche aufgrund der einfacheren Lesbarkeit aus gespiegelter Sichtweise initialisiert wird. Um für die Figur des Königs zwischen Mittel- und Endpiel unterscheiden zu können, existiert zusätzlich das Dictionary `PIECE_TO_PST_KING_ENDGAME`.\n",
```

File: src/Exercise03AI.ipynb [hunk 2]
```diff
@@ -130,8 +173,23 @@
...
-    "    ]\n",
-    "\n",
+    "    ]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Türme haben nur kleine Differenzen in der Bewertung. So stehen diese auf den Felden `d1` und `e1` sowie auf den 7. Linie (`a7` bis `h7`) besonders gut. An den Vertikalen am Rand (Linie `a` und `h`) stehen diese schlechter."
```
Comment (p-fruck) [inline src/Exercise03AI.ipynb:183]:
```suggestion
    "Türme haben nur kleine Differenzen in der Bewertung. So stehen diese auf den Feldern `d1` und `e1` sowie auf den 7. Linie (`a7` bis `h7`) besonders gut. An den Vertikalen am Rand (Linie `a` und `h`) stehen diese schlechter."
```

File: src/Exercise03AI.ipynb [hunk 3]
```diff
@@ -141,8 +199,23 @@
...
-    "    ]\n",
-    "\n",
+    "    ]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Die Dame kann durch ihre hohe Anzahl an Zug- und Schlag-Richtungen in der Mitte des Spiels sehr viel abdecken, sollte also dort ausgerichtet sein. Auch sie sollte weden am Rand noch in den Ecken plaziert werden."
```
Comment (p-fruck) [inline src/Exercise03AI.ipynb:209]:
```suggestion
    "Die Dame kann durch ihre hohe Anzahl an Zug- und Schlag-Richtungen in der Mitte des Spielfelds sehr viel abdecken, sollte also dort ausgerichtet sein. Auch sie sollte weder am Rand noch in den Ecken platziert werden."
```

File: src/Exercise03AI.ipynb [hunk 4]
```diff
@@ -152,8 +225,23 @@
...
-    "    ]\n",
-    "\n",
+    "    ]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Einzigst für den König gibt es zwei Piece-Square Tables. Die Folgende ist für das Mittelspiel, die Nächste für das Endspiel. Im Mittelspiel soll durch die Wertung verhindert werden, dass der König zu früh in die Mitte des Spielfelds zieht und von anderen Figuren angegriffen werden kann. Auf der Grundlinie ist er von seinen anderen Figuren geschützt."
```
Comment (p-fruck) [inline src/Exercise03AI.ipynb:235]: Bekanntes Schwabenproblem :laughing: 
```suggestion
    "Einzig für den König gibt es zwei Piece-Square Tables. Die Folgende ist für das Mittelspiel, die Nächste für das Endspiel. Im Mittelspiel soll durch die Wertung verhindert werden, dass der König zu früh in die Mitte des Spielfelds zieht und von anderen Figuren angegriffen werden kann. Auf der Grundlinie ist er von seinen anderen Figuren geschützt."
```

**Review by p-fruck — CHANGES_REQUESTED** [review]

**Review by antonplagemann — CHANGES_REQUESTED** [review]
Nice 🙌

File: src/Exercise03AI.ipynb [hunk 5]
```diff
@@ -20,7 +20,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%load_ext nb_black\n",
+    "#%load_ext nb_black\n",
```
[the remark this answers is not in the document]
Comment (piuswalter) [reply src/Exercise03AI.ipynb]: Fixed in 8354f68.

File: src/Exercise03AI.ipynb [hunk 6]
```diff
@@ -95,10 +93,25 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Nachgehend werden für alle Figuren die benötigten Piece-Square Tables definiert sowie kurz beschrieben.\n",
     "\n",
-    "    # Define field values as class constants\n",
-    "    # fmt: off\n",
+    "Anhand der Piece-Square Table der Bauern `PST_PAWN`, die im Folgenden definiert ist, wird ersichtlich, dass Bauern auf den Feldern `f2`, `g2` und `h2` (symmetrischer Weise natürlich auch auf `a2`, `b2` und `c2`) Boni (positive Bewertung) erhalten, während sie auf den Felder `f3` und `g3` Abzüge (negative Bewertungen) erhalten."
```
[the remark this answers is not in the document]
Comment (piuswalter) [reply src/Exercise03AI.ipynb]: Fixed in c7c0607.

File: src/Exercise03AI.ipynb (hunk 4 above)
[the remark this answers is not in the document]
Comment (piuswalter) [reply src/Exercise03AI.ipynb]: Fixed in c7c0607.

**Review by antonplagemann — APPROVED** [review]

**Review by p-fruck — APPROVED** [review]

**Pull request merged**